L32-opencensus-span-api.md

```diff
@@ -0,0 +1,59 @@
+OpenCensus C++ Span API changes
+----
+* Author(s): @g-easy
+* Approver: a11r
+* Status: Draft
+* Implemented in: C++
+* Last updated: 2018-07-17
+* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
Please create a thread and fill this in. This will have some discussion because of the Abseil issue.

Reply:
Done.